src/lcmap_rest/tile/db.clj

```diff
@@ -52,10 +52,11 @@
                                :x
                                :y
                                :acquired
+                               :source
                                :data)
         _       (cql/use-keyspace conn ks)
         results (cql/select conn table columns where)]
-    (map #(merge spec %) results)))
+    results))
```

Comment:
Probably don't need the last definition in let, yeah? We could just return ``(cql/select conn table columns where)`` ...